var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -555,6 +560,8 @@ def configure_args(self):
             device_config = "--with-device=ch4:"
         elif "device=ch3" in spec:
             device_config = "--with-device=ch3:nemesis:"
+        elif "device=ch3:sock" in spec:
+            device_config = "--with-device=ch3:sock:"
```

Comment:
```suggestion
            device_config = "--with-device=ch3:sock"
```
`ch3:sock` is the full device configuration string. Nothing after it.

Reply:
Thanks, will fix.